src/EditorFeatures/VisualBasicTest/UseInferredMemberName/UseInferredMemberNameTests.vb

```diff
@@ -39,6 +39,21 @@ End Class
 ", parseOptions:=s_parseOptions)
         End Function
 
+        <Fact>
+        <WorkItem(23659, "https://github.com/dotnet/roslyn/issues/23659")>
+        Public Async Function TestBug() As Task
```

Comment:
Nit: Test names like this are the worst - `TestMissingForObjectCreation` would be much better.

Reply:
My bad. Intended to fix, but forgot.